pkg/controller/sparkapplication/monitoring_config.go

```diff
@@ -44,14 +44,8 @@ func configPrometheusMonitoring(app *v1beta2.SparkApplication, kubeClient client
 		port = *app.Spec.Monitoring.Prometheus.Port
 	}
 
-	var javaOption string
-	if app.HasPrometheusConfigFile() {
-		configFile := *app.Spec.Monitoring.Prometheus.ConfigFile
-		glog.V(2).Infof("Overriding the default Prometheus configuration with config file %s in the Spark image.", configFile)
-		javaOption = fmt.Sprintf("-javaagent:%s=%d:%s", app.Spec.Monitoring.Prometheus.JmxExporterJar,
-			port, configFile)
-	} else {
-		glog.V(2).Infof("Using the default Prometheus configuration.")
+	if !app.HasMetricsPropertiesFile() || !app.HasPrometheusConfigFile() {
```

Comment:
Can we refactor the if-else block like the following to make it easier to read and understand?

```
var javaOption string

if app.HasMetricsPropertiesFile() && app.HasPrometheusConfigFile() {
    glog.V(2).Infof("Loading local metrics and Prometheus configuration files")
    ...
    javaOptions = ...
} else {
    glog.V(2).Infof("Creating a ConfigMap for metrics and/or Prometheus configurations")
    ...
    javaOptions = ...
}
```